appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -2397,18 +2397,18 @@ private static String collectUserProjectErrorInfo(final String userId, final lon
   @Override
   public String uploadTempFile(byte[] content) throws IOException {
     String uuid = UUID.randomUUID().toString();
-    String fileName = "__TEMP__/" + uuid;
+    String fileName = TEMP_PREFIX + "/" + uuid;
     setGcsFileContent(fileName, content);
     return fileName;
   }
 
   @Override
   public InputStream openTempFile(String fileName) throws IOException {
-    if (!fileName.startsWith("__TEMP__")) {
+    if (!fileName.startsWith(TEMP_PREFIX)) {
       throw new RuntimeException("deleteTempFile (" + fileName + ") Invalid File Name");
     }
     // Use FileData.RoleEnum.TARGET because these temp files never live very long
-    GcsFilename gcsFileName = new GcsFilename(getGcsBucketToUse(FileData.RoleEnum.TARGET), fileName);
+    GcsFilename gcsFileName = new GcsFilename(getGcsBucketToUse(FileData.RoleEnum.TEMPORARY), fileName);
```

Comment:
Note this did not have any impact, but worth to keep it with the expectation to prevent surprises in the future...